app/extensions.js

```diff
@@ -141,7 +141,7 @@ let generateBraveManifest = () => {
     'form-action': '\'none\'',
     'referrer': 'no-referrer',
     'style-src': '\'self\' \'unsafe-inline\'',
-    'img-src': '* data:',
+    'img-src': '\'self\' file://*',
```

Comment:
i think you need to keep '*' and 'data:', otherwise favicons won't load in about:preferences ledger panel